Realm/Realm/Realm.cs

```diff
@@ -114,7 +115,7 @@ public static Task<Realm> GetInstanceAsync(RealmConfigurationBase config = null)
                 schema = RealmSchema.Default;
             }
 
-            return config.CreateRealmAsync(schema);
+            return config.CreateRealmAsync(schema, cancellationToken ?? CancellationToken.None);
```

Comment:
```suggestion
            return config.CreateRealmAsync(schema, cancellationToken);
```
with the above suggestion